implementation/physicsengine/pom.xml

```diff
@@ -25,27 +26,22 @@
             </plugin>
 
             <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-            </plugin>
-
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-shade-plugin</artifactId>
-                <executions>
-                  <execution>
-                    <goals>
-                      <goal>shade</goal>
-                    </goals>
-                    <configuration combine.self="append">
-                        <transformers>
-                            <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
-                                <mainClass>org.inaetics.isep.PhysicsEngine</mainClass>
-                            </transformer>
-                        </transformers>
-                    </configuration>
-                  </execution>
-                </executions>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Export-Package>org.inaetics.dronessimulator.physicsengine,
+                                        org.inaetics.dronessimulator.common,
+                                        org.inaetics.dronessimulator.common.protocol,
+                                        org.inaetics.dronessimulator.pubsub.api,
+                                        org.inaetics.dronessimulator.pubsub.api.publisher,
+                                        org.inaetics.dronessimulator.pubsub.api.subscriber,
+                                        org.inaetics.dronessimulator.pubsub.api.serializer</Export-Package>
```

Comment:
Serializer package is not needed - only required by publisher and subscriber (Felix dependency manager)